src/components/UI/MyAccordion/MyAccordion.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
```

Comment:
Программа упала, не может найти '@material-ui/icons/ExpandMore':

![review1](https://user-images.githubusercontent.com/86306282/125266657-d9120a80-e327-11eb-9660-2e4facee1cc0.png)

Reply:
npm install выполнить надо. Установил в проект @material-ui/icons

Reply:
можно 4 строки объединить в одну
```suggestion
import { Accordion, AccordionSummary, AccordionDetails, Typography } from '@material-ui/core';
```

Reply:
Исправлен